mantidproject/mantid
# Updated Inelastic Data Analysis Documentation

**Description of work.**
Added an updated screenshot of the Ineleastic Data Analysis GUI to the Inelastic Data Analysis Documentation and the Elwin and I(Q,t) documentation. Also updated the Elwin documentation with details of the new loading facility including a gif demonstrating how to use.

**To test:**
- remove build/docs/doctrees
- build the documents again and check there are no warnings
- Check the information is still correct
- Are the images visible and clearly illustrate the new feature/improvement?
- Grammar/Spelling check


Fixes #32364

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
